dataRetriever/requestHandlers/requestHandler.go

```diff
@@ -242,7 +242,7 @@ func (rrh *resolverRequestHandler) RequestShardHeader(shardID uint32, hash []byt
 		return
 	}
 
-	log.Trace("requesting shard header from network",
+	log.Debug("requesting shard header from network",
```

Comment:
Is the Debug needed?

Reply:
yes, for the moment, we want to see the effective requests

Reply:
ok